database/seed/schema.sql

```diff
@@ -0,0 +1,29 @@
+DROP DATABASE IF EXISTS stock_chart;
+
+CREATE DATABASE stock_chart;
+
+DROP TABLE IF EXISTS stock_quotes;
+DROP TABLE IF EXISTS stock_info;
+
+\connect stock_chart;
+
+CREATE TABLE IF NOT EXISTS stock_info (
+ id INTEGER PRIMARY KEY,
+ ticker VARCHAR (5),
+ average_stock NUMERIC (5, 2),
+ change_percent NUMERIC (3, 2),
+ company VARCHAR (100),
+ num_owners INTEGER,
+ recommendation_percent NUMERIC (3, 2),
+ related_tags TEXT []
+);
+
+CREATE TABLE IF NOT EXISTS stock_quotes (
+ stock_id INTEGER REFERENCES stock_info(id),
+ price NUMERIC (6, 2),
```

Comment:
Good choice. [NUMERIC type](http://www.postgresqltutorial.com/postgresql-numeric/) is better than FLOAT type for monetary values. Maybe also consider [data validation](https://www.postgresql.org/docs/9.4/ddl-constraints.html) for invalid or overly large inputs and taking advantage of the ability of NUMERIC type to store 'NaN' versus null for default values.